docs/tutorials/tutorial_polymers.rst

```diff
@@ -0,0 +1,332 @@
+            mb.translate(self['down'], [0, 0.07, 0]) 
+            mb.rotate_around_y(self['down'], 3.14) 
+            mb.rotate_around_x(self['down'], -0.59) 
+    
+    
+    
+    class AlkanePolymer(mb.Compound):
+        def __init__(self, chain_length=1):
+            super(AlkanePolymer, self).__init__()
+    
+            last_monomer = CH2()
+            self.add(last_monomer)
+    
+            for i in range (chain_length-1):
+                current_monomer = mb.clone(last_monomer)
+        
+                mb.force_overlap(move_this=current_monomer, 
+                                 from_positions=current_monomer['up'], 
+                                 to_positions=last_monomer['down'])
+                self.add(current_monomer)
+                last_monomer=current_monomer
+
+.. code:: ipython3
+
+    polymer = AlkanePolymer(chain_length = 10)
+    polymer.visualize(show_ports=True)
+
+Using mBuild's Polymer Class
+----------------------------
+
+mbuild provides a prebuilt class to perform this basic functionality.
+Since it is designed to be more general, it takes as an argument not
+just the chain length, but also the monomer and the port labels (i.e.,
+'up' and 'down', since these labels are user defined).
+
+.. code:: ipython3
+
+    polymer = mb.Polymer(CH2(), 10, port_labels=('up', 'down'))
+    polymer.visualize()
+
+Building a System of Alkanes
+----------------------------
+
+A system of alkanes can be constructed by simply cloning the polymer
+constructed above and translating the alkanes in space (and/or rotating
+them). mBuild provides many routines that can be used to create
+different patterns, to which the polymers can be shifted to.
+
+.. code:: ipython3
+
+    #define a compound to hold all the polymers
+    system = mb.Compound()
+    
+    
+    #we will first rotate the chain such that the long dimension is oriented along the z-axis
+    mb.rotate_around_x(polymer, 3.14/2.0)
+    
+    pattern_disk = mb.DiskPattern(50)
+    
+    #patterns are generated betwee 0 and 1, 
+    #and thus need to be scaled to provide appropriate spacing
+    pattern_disk.scale(5) 
+    
+    for pos in pattern_disk:
+        
+        current_polymer = mb.clone(polymer)
+        mb.translate(current_polymer, pos)
+        system.add(current_polymer)
+
+.. code:: ipython3
+
+    system.visualize()
+
+Other patterns can be used, e.g., the Grid3DPattern. We can also use the
+rotation commands to randomize the orientation.
+
+.. code:: ipython3
+
+    #define a compound to hold all the polymers
+    system = mb.Compound()
+    import random
+    
+    #we will first rotate the chain such that the long dimension is oriented along the z-axis
+    mb.rotate_around_x(polymer, 3.14/2.0)
+    
+    pattern_disk = mb.Grid3DPattern(5,5,5)
+    
+    
+    for pos in pattern_disk:
+        
+        #scale the pattern 
+        pos[0] = pos[0]*8.0
+        pos[1] = pos[1]*8.0
+        pos[2] = pos[2]*8.0
```

Comment:
This initially had each direction scaled differently (since the scale function only took a scalar).